src/openrct2/management/NewsItem.h

```diff
@@ -241,7 +241,10 @@ namespace OpenRCT2::News
 
         void clear() noexcept
         {
-            front().Type = News::ItemType::Null;
+            for (size_t i = 0; i < N; i++)
+            {
+                Queue[i].Type = News::ItemType::Null;
```

Comment:
Perhaps we could change the constructor for News::Item to default to `News::ItemType::Null` instead?